cmd/operator/main.go

```diff
@@ -62,33 +62,12 @@ func main() {
 		publicNamespace = flag.String("public-namespace", operator.DefaultPublicNamespace,
```

Comment:
It's certainly a more involved change now. However, it seems overall consistent: if you change where you deploy the manifests into, you also update the namespace flags in the operator YAML. And as you say, this would just be templated.

Added the explicit flag values to the operator.